posthog/api/capture.py

```diff
@@ -443,6 +443,7 @@ def capture_internal(event, distinct_id, ip, site_url, now, sent_at, event_uuid=
     kafka_partition_key = None
 
     if event["event"] in ("$snapshot", "$performance_event"):
+        kafka_partition_key = f"{token}:{event['properties']['$session_id']}"
```

Comment:
Only nastiness here is that the partition will change if the team token changes... Wondering - do we actually care about the team token? It's not important really to the consumption logic

Reply:
consider it gone